packages/components/src/components/alert/readme.md

```diff
@@ -7,12 +7,27 @@
 
 ## Properties
 
-| Property      | Attribute      | Description | Type     | Default |
-| ------------- | -------------- | ----------- | -------- | ------- |
-| `customClass` | `custom-class` |             | `string` | `''`    |
-| `size`        | `size`         |             | `string` | `''`    |
-| `theme`       | `theme`        |             | `string` | `''`    |
-| `variant`     | `variant`      |             | `string` | `''`    |
+| Property      | Attribute      | Description | Type      | Default     |
+| ------------- | -------------- | ----------- | --------- | ----------- |
+| `customClass` | `custom-class` |             | `string`  | `''`        |
```

Comment:
Please consider adding `description` to the props, see `button` component for an example